apps/api/migrations/108_create_balance_audit_log.up.sql

```
@@ -0,0 +1,65 @@
+-- balanceaudit.SystemActor.
+INSERT INTO balance_audit_log (
+    vault_id, user_id, actor, operation, amount, balance_before, balance_after, created_at
+)
+SELECT
+    v.id,
+    v.user_id,
+    'system:migration',
+    'opening_balance',
+    v.current_balance,
+    0,
+    v.current_balance,
```

SQLFluff (4.3.0)

[error] Reuse of column alias 'current_balance' from line 58.

(AL08)